### PR TITLE
Add possibility to enable ccache to the build script

### PR DESCRIPTION
**Pull request by Blackhex**
Usefull for local builds only, for GHA we'd have to setup sccache in cloud.

Some results how ccache affects the build time:
```
without ccache

real    4m54.705s
user    65m49.265s
sys     4m44.815s

ccache fist run

real    5m13.932s
user    70m52.523s
sys     5m30.535s

ccache second run

real    3m1.360s
user    11m30.533s
sys     1m54.457s
```

### CODE REVIEW

File: .github/scripts/build.sh [hunk 1]
```diff
@@ -2,6 +2,17 @@
 
 source `dirname ${BASH_SOURCE[0]}`/config.sh
 
+if [ "$CCACHE" = 1 ] ; then
+  pushd /usr/lib/ccache
+    if ! [ -f $TARGET-gcc ]; then
+      sudo ln -s ../../bin/ccache $TARGET-gcc
```
Comment (eukarpov) [inline .github/scripts/build.sh:8]: Is there any other way to avoid sudo call?

Comment (Blackhex) [reply .github/scripts/build.sh:8]: Yes, create `aarch64-w64-mingw32` symlinks in custom directory and add that to the `PATH`.

Comment (Blackhex) [reply .github/scripts/build.sh:8]: Yes, that's what I had in mind (if you mean `PATH=/usr/lib/ccache/:$TOOLCHAIN_PATH/ccache:$TOOLCHAIN_PATH/bin:$PATH`).

Comment (eukarpov) [reply .github/scripts/build.sh:8]: looks good to me

**Review by eukarpov — APPROVED** [review]

**Pull request merged**